fuse/utils/ndict.py

```diff
@@ -144,7 +144,7 @@ def values(self) -> dict_items:
     def items(self) -> dict_items:
         return self._stored.items()
 
-    def merge(self, other: NDict) -> None:
+    def merge(self, other: MutableMapping) -> None:
```

Comment:
Now that I see it:

Why do we even need `merge()` if we just use the `update()` (which compatible regular dict..)

Reply:
In the same context, I've send you a cool library on Slack that might be a cool replacement for the loved `NDict`

Reply:
🤷 
Lets keep it for backward compatibility though
Saw it,
We started with this library btw, but then we decided to move to NDict cause it was with too many capabilities for our needs.

Reply:
Cool! good to know! 

Indeed the `NDict` is pretty slim and simple (and thus fast).